deploy/examples/operator-openshift.yaml

```diff
@@ -179,10 +179,10 @@ data:
   # ROOK_CSI_NFS_IMAGE: "mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0"
 
   # (Optional) set user created priorityclassName for csi plugin pods.
-  # CSI_PLUGIN_PRIORITY_CLASSNAME: "system-node-critical"
+  CSI_PLUGIN_PRIORITY_CLASSNAME: "system-node-critical"
 
   # (Optional) set user created priorityclassName for csi provisioner pods.
-  # CSI_PROVISIONER_PRIORITY_CLASSNAME: "system-cluster-critical"
+  CSI_PROVISIONER_PRIORITY_CLASSNAME: "system-cluster-critical"
```

Comment:
@travisn this is missing in helm templates we might need to add it as well to make it complete?

Reply:
Need to enable here https://github.com/rook/rook/blob/24226659cc98a890abfb31c0183ac38ae2af5a49/deploy/charts/rook-ceph/values.yaml#L74-L77

Reply:
Good catch, will add it